spec/support/potassium_test_helpers.rb

```diff
@@ -73,10 +71,26 @@ def hash_to_arguments(hash)
     end.join(" ")
   end
 
+  def add_to_path(new_path, append = false)
+    ENV['PATH'] = append ? "#{ENV['PATH']}:#{new_path}" : "#{new_path}:#{ENV['PATH']}"
+  end
+
+  def project_bin
+    File.join(project_path, 'bin')
+  end
+
+  def potassium_bin
+    File.join(root_path, "bin", "potassium")
+  end
```

Comment:
Given that now the whole bin is in the PATH, I think it's not necessary to call `bin/potassium` anymore

Reply:
While I share those concerns, in this case the added path is absolute (not just `./bin`), which is a bit safer. Also, we are only doing this for the spec run, with the dummy app, so I wouldn't worry to much about it since we are in control with what is installed and what is not.

Reply:
You are right!

Reply:
Got confused there, you're right!